### ISSUE TITLE
Issues with symbol-observable: Property 'observable' must be of type 'symbol'

### ISSUE THREAD

**Issue by muuvmuuv** [angular/angular-cli#19411]
# 🐞 Bug report

### Command (mark with an `x`)

- [ ] new
- [ ] build
- [x] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] extract-i18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

 Yes, the previous version in which this bug was not present was: ....

Yes and no. Depending which other lib I use forgot to update to a newer `symbol-observable` version.

### Description

 A clear and concise description of the problem...

This is something I get into more and more. One package depending on version x and the other on version y of the same lib. But this lib is not allowing different typed versions or in this case it is not even clear why different Observable approaches are used.

I have the following installed:

```
├─┬ @angular/cli@11.0.1
│ └── symbol-observable@2.0.3
├─┬ @apollo/client@3.2.6
│ └── symbol-observable@3.0.0
```

Running `serve` prints out:

```
ERROR in node_modules/symbol-observable/index.d.ts:6:14 - error TS2717: Subsequent property declarations must have the same type.  Property 'observable' must be of type 'symbol', but here has type 'unique symbol'.

6     readonly observable: unique symbol;
               ~~~~~~~~~~

  node_modules/rxjs/internal/symbol/observable.d.ts:4:18
    4         readonly observable: symbol;
                       ~~~~~~~~~~
    'observable' was also declared here.
```

## 🔬 Minimal Reproduction

Its hard to find where the problem come from, since the TypeScript nor Angular-CLI prints any line where that type issue come from.
So before I take that time to create a repl I would like to hear if this is an easy fix and if this `symbol-observable` is still needed because I thought RxJS provides such already.

## 🔥 Exception or Error

```
ERROR in node_modules/symbol-observable/index.d.ts:6:14 - error TS2717: Subsequent property declarations must have the same type.  Property 'observable' must be of type 'symbol', but here has type 'unique symbol'.

6     readonly observable: unique symbol;
               ~~~~~~~~~~

  node_modules/rxjs/internal/symbol/observable.d.ts:4:18
    4         readonly observable: symbol;
                       ~~~~~~~~~~
    'observable' was also declared here.
```

## 🌍 Your Environment

```
Angular CLI: 11.0.1
Node: 14.15.1
OS: darwin x64

Angular: 11.0.0
... animations, cdk, common, compiler, compiler-cli, core, forms
... material, platform-browser, platform-browser-dynamic, router
... service-worker
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1002.0
@angular-devkit/build-angular   0.1100.1
@angular-devkit/core            10.2.0
@angular-devkit/schematics      11.0.1
@angular/cli                    11.0.1
@schematics/angular             11.0.1
@schematics/update              0.1100.1
rxjs                            6.6.2
typescript                      4.0.5
```

**Anything else relevant?**

**Comment by alan-agius4** [comment]
Hi,

The `symbol-observable` package is a ponyfill for `Symbol.observable` https://github.com/tc39/proposal-observable

The issue here seems to be related to https://github.com/benlesh/symbol-observable/issues/41 which was released as `3.0.0` and is needed by `@apollo/client`.

Closing as this doesn't look like an Angular issue.

**Comment by muuvmuuv** [comment]
With angular version `11.0.1` and apollo-client version `3.2.7` it got fixed.
